Qt/devices/deviceinfo.json

```
@@ -1,9 +1,18 @@
 {
-    "basedevice":{
+    "basedevice": {
         "Name": "Generic Device",
         "MaxNameLength": 24,
         "HiddenFeatures": []
     },
+    "w820": {
```

Thanks! Please use a similar name style here.
```suggestion
    "w820nb": {
```